muxers/mplex/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.20.0 [2020-06-30]
+
+- Updated dependencies.
```

Comment:
Mplex (and yamux) changed their APIs to adapt to the changes in core, e.g. renamed `StreamMuxer::poll_inbound` to `StreamMuxer::poll_event` and return an event instead of just inbound substreams.

Reply:
Yes, these are the details of what it means to update the `libp2p-core` dependency. If you think it is worthwhile to be a bit more verbose here, I can do that!

Reply:
See https://github.com/libp2p/rust-libp2p/pull/1640/commits/49c964fc591cb24f0f76ae51c9393355a489a4c2.